x-element.js

```diff
@@ -21,6 +22,24 @@ export default class XElement extends HTMLElement {
     return XElement.defaultTemplateEngine;
   }
 
+  /**
+   * Declare an initialization object for the host’s shadow root.
+   * See https://developer.mozilla.org/en-US/docs/Web/API/Element/attachShadow.
+   */
+  static get shadowRootInit() {
```

Comment:
We’ve talked about pulling this chunk of configuration into a declarative getter before — seems much more reasonable now that we have _multiple_ things to configure when creating the render root.